Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.py

```diff
@@ -0,0 +1,3105 @@
+    file_paths = argToList(args.get('file_path'))
+    all_files_response = []
+    for file_path in file_paths:
+        parameters = {'file_path': file_path}
+        response = client.run_script('548023b6e4a01ec51a495ba6e5d2a15d', endpoint_ids, parameters, timeout, incident_id)
+        reply = response.get('reply')
+        all_files_response.append(CommandResults(
+            readable_output=tableToMarkdown(f'Run Script Delete File on {file_path}', reply),
+            outputs_prefix=f'{INTEGRATION_CONTEXT_BRAND}.ScriptRun',
+            outputs_key_field='action_id',
+            outputs=reply,
+            raw_response=reply,
+        ))
+    return all_files_response
+
+
+def run_script_file_exists_command(client: Client, args: Dict) -> List[CommandResults]:
+    endpoint_ids = argToList(args.get('endpoint_ids'))
+    incident_id = arg_to_number(args.get('incident_id'))
+    timeout = arg_to_number(args.get('timeout', 600)) or 600
+    file_paths = argToList(args.get('file_path'))
+    all_files_response = []
+    for file_path in file_paths:
+        parameters = {'path': file_path}
+        response = client.run_script('414763381b5bfb7b05796c9fe690df46', endpoint_ids, parameters, timeout, incident_id)
+        reply = response.get('reply')
+        all_files_response.append(CommandResults(
+            readable_output=tableToMarkdown(f'Run Script File Exists on {file_path}', reply),
+            outputs_prefix=f'{INTEGRATION_CONTEXT_BRAND}.ScriptRun',
+            outputs_key_field='action_id',
+            outputs=reply,
+            raw_response=reply,
+        ))
+    return all_files_response
+
+
+def run_script_kill_process_command(client: Client, args: Dict) -> List[CommandResults]:
+    endpoint_ids = argToList(args.get('endpoint_ids'))
+    incident_id = arg_to_number(args.get('incident_id'))
+    timeout = arg_to_number(args.get('timeout', 600)) or 600
+    processes_names = argToList(args.get('process_name'))
+    all_processes_response = []
+    for process_name in processes_names:
+        parameters = {'process_name': process_name}
+        response = client.run_script('fd0a544a99a9421222b4f57a11839481', endpoint_ids, parameters, timeout, incident_id)
+        reply = response.get('reply')
+        all_processes_response.append(CommandResults(
+            readable_output=tableToMarkdown(f'Run Script Kill Process on {process_name}', reply),
+            outputs_prefix=f'{INTEGRATION_CONTEXT_BRAND}.ScriptRun',
+            outputs_key_field='action_id',
+            outputs=reply,
+            raw_response=reply,
+        ))
+
+    return all_processes_response
+
+
+def add_exclusion_command(client: Client, args: Dict) -> CommandResults:
+    name = args.get('name')
+    if not name:
+        raise DemistoException("Didn't get name arg. This arg is required.")
+    indicator = args.get('filterObject')
+    if not indicator:
+        raise DemistoException("Didn't get filterObject arg. This arg is required.")
```

Comment:
why don't we config them as required in the yml?

Reply:
We config, I wrote the errors when I tested with demistomock, can remove them

Reply:
Removed

Reply:
After removal, mypy is angry on me because of the json.loads after it, so I added